<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWidget</class>
 <widget class="QWidget" name="WelcomeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>579</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>C3D</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Open recent</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget::item:hover { 
	background-color: rgba(63, 191, 202, 0.5);
}</string>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonCreate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>54</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align:left;
padding-left: 1em;
padding-right: 1em;</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string>Create new project</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/resources.qrc">
             <normaloff>:/res/icons/editor/newfile.svg</normaloff>:/res/icons/editor/newfile.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonOpen">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>54</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align:left;
padding-left: 1em;
padding-right: 1em;</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string>Open a local folder</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/resources.qrc">
             <normaloff>:/res/icons/editor/open.svg</normaloff>:/res/icons/editor/open.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonTutorialGallery">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>54</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align:left;
padding-left: 1em;
padding-right: 1em;</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string>Open tutorials gallery</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/resources.qrc">
             <normaloff>:/res/icons/gallery.svg</normaloff>:/res/icons/gallery.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDocumentation">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>54</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align:left;
padding-left: 1em;
padding-right: 1em;</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string>Open Documentation</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/resources.qrc">
             <normaloff>:/res/icons/doc.svg</normaloff>:/res/icons/doc.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonManuals">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>54</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align:left;
padding-left: 1em;
padding-right: 1em;</string>
           </property>
           <property name="text">
            <string>Open Manuals</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/resources.qrc">
             <normaloff>:/res/icons/manuals.svg</normaloff>:/res/icons/manuals.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QComboBox" name="comboBoxThemes">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelThemes">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Theme</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxLanguages">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>RU</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ENG</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelLanguages">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Language</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
